Req Supplypacks Overhaul (Operators Operating Operationally)

Alterist opened the pull request:
Corrected a few typos in the list
Moved the Pyro's flamer fuel to Ammo 
Added the Semiauto Shotgun to the list of buyable crates (shotgun x1 all ammo x1 for 60 points)
Added c4 to the list of buyable crates (x5 for 60 points)
Added auto-shotty stocks to the stock attachment crate, reduced amount to 2 stocks of each type
Added Binos and Tactical Binos to the list of buyables
Separated the Scout's M4RA ammo into 3 different buyable crates
Changed the costume closets into crates (Since closets are all anchored, it's impossible to move them off the cargo pad)
Matched the actual amount of flamers and flamer tanks in a crate to what it should be according to it's description(+1 flamethrower per crate, +3 tank per crate)
Added /improper to all of the crate names and corrected a few so that it is clearer and more consistent
Made the spec-ops crate into something interesting instead of being a joke, increased it's price

Iatots commented:
Only problem I see is TCs ordering spec ops crate every round to make sniping with the tank an absolute nightmare for benos. They already drained req every round for modules, ROs won't deny tanks one crate to make them ++% more effective.

Scout spec impact ammo seems to cost as much as regular ammo. Yeah, it's a bit less precise and has lower armor penetration but stuns are really strong. They are honestly the ammo of choice if you are not running a stealth double BC buckshot meme, I'd say up their price to that of incendiary.

Ghommie commented:
I think that this spec crate, albeit being less of a joke compared to the previous, it has the potentials to be a waste of req's credits just to buff a handful of marines or less. On my opinion, switch NVGs out for else, bring back the Swat masks while giving them 0 vision impair, and maybe 1 anti_hug, bring the turtlenecks' stats up to par with marine suit, and restore the original price.

Kingtrin commented:
Yea NVG's are probably too much unless we can figure out some way to nerf its effectiveness with regards to tank crews. I see the spec ops crate being still something of a meme if you remove tank crew from the equation, because when I think of the person ordering the crate (and being given a yes) its a naked CO with a machete thirsty for beno ass.

Alterist commented:
i forgot about the Night Vision and Artillery mod interaction, i'll remove that

Alterist commented on code/datums/supplypacks.dm:
```diff
@@ -1183,21 +1224,64 @@ AMMO
+					/obj/item/ammo_magazine/rifle/m4ra/incendiary,
+					/obj/item/ammo_magazine/rifle/m4ra/incendiary
+					)
+	cost = RO_PRICE_NORMAL
```
so having 5 mags for NORMAL price is too cheap for the M4RA, while it's ok for the sniper to have 6 for NORMAL price?

Rohesie replied:
The sniper shoots much, much slower and is very much support. Besides IFF and less scarce special ammo the sniper has nothing on the scout, who is essentially a better sniper with much higher DPS and a stunning ammo. The most lethal marine really, when going solo. The same cannot be said for the sniper.

Alterist commented:
Increased price of binos by 2 stages each
M4RA special ammo crates now hold 3 each
Combat shotgun crates contains 1 of each type of ammo
Deleted duplicates in surplus ammo crate
Added really cheap empty crates with each squad's colors, intended for supply drops
Created a special tranq pistol for the spec ops crate

Alterist commented:
Ammo Amount on M4RA Ammo Crates fixed
man, I don't even know what happened to the point blank bit, it's fixed

Alterist commented:
adjusted Tactical Bino crate cost and amount
Adjusted Surgical crate cost and amount
Adjusted Spec-op crate cost and amount
